mandatory/2-function-creation.js

```diff
@@ -15,26 +17,67 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  let discount = 0;
+  if (totalPrice > 200) {
+    discount = 0.1;
+  } else if (totalPrice < 200) {
+    discount = 0.05;
+  }
+  return totalPrice - totalPrice * discount;
+}
+
+// or
+
+// function applyDiscount(totalPrice) {
+//   if (totalPrice > 200){
+//     return totalPrice * 0.9;
+//   } else {
+//     return totalPrice * 0.95;
+//   }
+// }
```

Comment:
Fab. I definitely prefer the code you chose over the commented out solution. Can you guess why?

Reply:
Regarding my approach to these tasks, I set aside my coding skills and knowledge and write the possible logical solutions using sudo language. Then, I use code to implement them. 

Back to your - why question -  I feel that the first solution is the most suitable since it is easy to understand, clear, and allows for the implementation of different discounts if needed.